setup.sh

```diff
@@ -0,0 +1,214 @@
+#!/bin/bash
+
+ARCH=$(uname -m)
+LATEST_RELEASE_VERSION=$(curl -s "https://api.github.com/repos/paradedb/paradedb/releases/latest" | jq -r .tag_name)
+LATEST_RELEASE_VERSION="${LATEST_RELEASE_VERSION#v}"
```

Comment:
Maybe we should name these differently? 

LATEST_RELEASE_TAG
LATEST_RELEASE_VERSION

?